app/javascript/app/components/emission-pathways-model-table/emission-pathways-model-table-component.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Table from 'components/table';
+import NoContent from 'components/no-content';
+import Loading from 'components/loading';
+import layout from 'styles/layout.scss';
+import EspModelsProvider from 'providers/esp-models-provider';
+import EspScenariosProvider from 'providers/esp-scenarios-provider';
+import EspIndicatorsProvider from 'providers/esp-indicators-provider';
+import styles from './emission-pathways-model-table-styles.scss';
+
+class EmissionPathwaysModelTableComponent extends PureComponent {
+  getTableContent() {
+    const { loading, data, noContentMsg, titleLinks } = this.props;
+    if (loading) return <Loading light className={styles.loader} />;
+    return data && data.length > 0 ? (
+      <Table
+        titleLinks={titleLinks}
+        data={data}
+        rowHeight={60}
+        sortBy={'name'}
+      />
+    ) : (
+      <NoContent message={noContentMsg} />
+    );
+  }
+
+  render() {
+    return (
+      <div className={layout.content}>
+        <EspModelsProvider />
```

Comment:
We could put these just on the parent page, instead of the table as they are needed for both?